Backend Code/VigyBag/VigyBag/settings.py

```diff
@@ -123,3 +176,9 @@
 # https://docs.djangoproject.com/en/3.2/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+#CORS_ALLOW_ALL_ORIGINS: True
+CORS_ALLOWED_ORIGINS = [
+    "http://localhost:5500",
+    
```

Comment:
I guess,  this is for your local setup.

Reply:
Yes, we can add other origins as well when required